en/release_note/release_note_latest_ver.rst

```diff
@@ -24,70 +26,108 @@ For CUBRID 9.3, please find https://www.cubrid.org/manual/en/9.3.0/release_note/
 Overview
 ========
 
-CUBRID 11.0 is the latest stable version that includes new features, significant changes and enhancements.
+CUBRID 11.2 is the latest stable version that includes new features, significant changes and enhancements.
 
 .. TODO: UPDATE WITH DETAILS.
 
-CUBRID 11.0
+CUBRID 11.2
 
-* is a version with improved security.
+* is a version with improved connectivity.
 * is more stable, faster, and more convenient for administrators.
 * fixes a large number of critical bugs.
-* includes useful SQL extensions: Supporting RVC (Row Value Constructor) and various REGEXP functions.
+* includes useful SQL extensions: User Schema concept and Synonym are supported.
 * includes code refactoring and modernization.
 
-CUBRID 11.0 **improves security** by providing data encryption and packet encryption. This version prevents abnormal data loss by supporting table-based TDE (Transparent Data Encryption) and packet encryption between the driver and server.
+CUBRID 11.2 **improved connectivity** by providing database link between homogeneous and heterogeneous DBMS. In addition, this version provides CDC (Change data capture) function by supporting supplemental logging that can track changed data.
+
+CUBRID 11.2 is **faster**. This version improves query optimization such as predicate pushdown and view transformation, supports Hash List Scan for all subqueries, improves performance of count(*) without conditional clause using cache, and improves truncate table to further improve performance.
```

Comment:
From 'without' to the end of the sentence, it is shown in italic style. Is it intended?

Reply:
The reason is unknown, but it is shown as italic style only in the code view. In 'view file', it is displayed normally.

Reply:
reflected.